rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -249,6 +249,50 @@ def testQuery_noSummaries(self):
     response = self.send_get('ParticipantSummary')
     self.assertBundle([], response)
 
+  def test_last_modified_sync(self):
+    num_participants = 20
+    SqlTestBase.setup_codes([PMI_SKIP_CODE], code_type=CodeType.ANSWER)
+    questionnaire_id = self.create_demographics_questionnaire()
+
+    # generate participants to count
+    for _ in range(num_participants):
+      # Set up participant, questionnaire, and consent
+      participant = self.send_post('Participant', {"providerLink": [self.provider_link]})
+      participant_id = participant['participantId']
+      with FakeClock(TIME_1):
+        self.send_consent(participant_id)
+      # Populate some answers to the questionnaire
+      answers = {
+        'race': RACE_WHITE_CODE,
+        'genderIdentity': PMI_SKIP_CODE,
+        'firstName': self.fake.first_name(),
+        'middleName': self.fake.first_name(),
+        'lastName': self.fake.last_name(),
+        'zipCode': '78751',
+        'state': PMI_SKIP_CODE,
+        'streetAddress': '1234 Main Street',
+        'city': 'Austin',
+        'sex': PMI_SKIP_CODE,
+        'sexualOrientation': PMI_SKIP_CODE,
+        'phoneNumber': '512-555-5555',
+        'recontactMethod': PMI_SKIP_CODE,
+        'language': PMI_SKIP_CODE,
+        'education': PMI_SKIP_CODE,
+        'income': PMI_SKIP_CODE,
+        'dateOfBirth': datetime.date(1978, 10, 9),
+        'CABoRSignature': 'signature.pdf',
+      }
+      self.post_demographics_questionnaire(participant_id, questionnaire_id, **answers)
+
+    response = self.send_get('ParticipantSummary?_sync=true&_count=10&lastModified=gt%s' % TIME_1)
```

Comment:
I'm not sure this is testing the `_sync` param.  We make 20 people, then get two pages of 10 each - so far we're just using the pagination token, if I understand correctly we have to make some more people and then use the link provided by the final page of the paginated resource.  So we could

  * make 10 people
  * get a page of 20 people - it will not have a next page, so the token will be for syncing
  * make 10 more people
  * use the URL given by the last page and verify that we got the second batch of ten people in return

Reply:
continuing to discuss offline, as there is an un-identified bug right now.